confd/templates/atdd-staging/kilda.properties.tmpl

```diff
@@ -27,4 +27,7 @@ reroute.delay={{ getv "/kilda_reroute_throttling_delay_min" }}
 reroute.hardtimeout={{ getv "/kilda_reroute_throttling_delay_max" }}
 discovery.interval={{ getv "/kilda_discovery_interval" }}
 discovery.timeout={{ getv "/kilda_discovery_timeout" }}
+antiflap.min={{ getv "/kilda_port_up_down_throttling_delay_seconds_min" }}
```

Comment:
cosmetics: would be nice to rename 'min' to something like
antiflap.throttling_delay

Reply:
All 3 of them are `throttling_delay`s. I mean if you read all three vars they only differ by 'min'-'warm_up'-'coold_down'